README.md

```diff
@@ -243,3 +244,23 @@ jobs:
           apiToken: ${{ secrets.CF_API_TOKEN }}
           accountId: ${{ secrets.CF_ACCOUNT_ID }}
 ```
+### "[NOTICE] Since you have specified an environment you need to make sure to pass in '--env dev' to your command."
+
+If you are setting [environment specific parameters](https://developers.cloudflare.com/workers/wrangler/environments/) via the `wrangler.toml` file. You need to ensure the environment parameter is passed to the `action` as well as in the `command` you are asking Wrangler to execute. This is becuase other Wrangler, such as secret uploads, need to know the environment context as well.
```

Comment:
```suggestion
If you are setting [environment specific parameters](https://developers.cloudflare.com/workers/wrangler/environments/) via the `wrangler.toml` file. You need to ensure the environment parameter is passed to the `action` as well as in the `command` you are executing. This is because other Wrangler commands, such as secret uploads, need to know the environment context as well.
```